packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java

```diff
@@ -193,6 +196,14 @@ private void updateJsMode(int mode) {
     }
   }
 
+  private void updateZoomMode(boolean mode) {
+    //webView.getSettings().setLoadWithOverviewMode(true);
+    webView.getSettings().setUseWideViewPort(mode);
```

Comment:
Why is this tied to the zoom mode?

Reply:
When the value is true and the page contains the viewport meta tag, the value of the width specified in the tag is used. If the page does not contain the tag or does not provide a width, then a wide viewport will be used, so you could always leave it on. do you suggest creating another parameter?